src/ml/neural_net/mps_layer_instance_norm_data_loader.mm

```diff
@@ -175,10 +177,11 @@ - (void) loadBeta:(float *)beta {
 
 - (float *) beta {
   NSUInteger previousStyle = _styleIndex;
+  _betaPlaceHolder = [NSMutableData data];
```

Comment:
Why do we need to allocate a new instance here? Don't we allocate in the data loader's initializer?

Note that you could do `_betaPlaceHolder.length = 0;` to clear out any current contents (and reuse the existing buffer)